src/scss/components/ec-grid/ec-grid.scss

```diff
@@ -0,0 +1,92 @@
+$gutters: 24px;
+$number-of-columns: 12;
+
+@mixin common {
+  display: flex;
+  justify-content: flex-start;
+  width: calc(100% + #{$gutters});
+  margin-left: calc(#{$gutters} / -2);
```

Comment:
All these call should be resolved in SASS, we don't need unnecessary calc() to do `calc(24px / 2)` when we could do
```
margin-left: $gutters / -2;
```